.vscodeignore

```diff
@@ -1,2 +1,16 @@
 .vscode/**
 .gitignore
+node_modules
+tsconfig.json
+
+# Client
+client/.gitignore
+client/node_modules
```

Comment:
If this behaves like `.gitignore` I would expect these two to be picked up by the earlier ones.